assets/js/apps/search/components/filters/filters-section/FiltersSection.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { Fragment } from 'react';
+import Tabs from "react-bootstrap/Tabs";
+import Tab from 'react-bootstrap/Tab';
+import { OBJECT_TYPE_STICKERS } from '../../TabStickers/TabSticker'
+import TypeSchemaList from '../type-schema-list/TypeSchemaList';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933948)
```suggestion
import TypeSchemaList from "../type-schema-list/TypeSchemaList";
```